ArduPlane/Parameters.cpp

```diff
@@ -1240,8 +1240,35 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("AUTOTUNE_AXES", 34, ParametersG2, axis_bitmask, 7),
 
+    // @Param: PROP_DSKLD
+    // @DisplayName: Propeller disk loading
+    // @Description: This is the vehicle weight in kg divided by the total disk area of all propellers in m^2. It is used to calculate the effect of prop wash onc ontrol surfaces. The effect on individual control surfaces is controlled by the PROP_RLL_COMP, PROP_PIT_COMP and PROP_YAW_COMP parameters.
+    // @Units: kg/m/m
+    // @Range: 0 50
+    // @User: Standard
+    AP_GROUPINFO("PROP_DSKLD", 35, ParametersG2, prop_disc_loading, 0),
+
+    // @Param: PROP_RLL_COMP
+    // @DisplayName: Propwash roll compensation
+    // @Description: This is a scaling factor applied to the theoretical calculation of aileron effectiveness variation with propeller thrust. The PROP_DSKLD parameter must be set for this to work,
+    // @Range: 0 1
+    // @User: Standard
+    AP_GROUPINFO("PROP_RLL_COMP", 36, ParametersG2, propwash_roll_comp, 0),
+
+    // @Param: PROP_PIT_COMP
+    // @DisplayName: Propwash roll compensation
+    // @Description: This is a scaling factor applied to the theoretical calculation of elevator effectiveness variation with propeller thrust. The PROP_DSKLD parameter must be set for this to work,
+    // @Range: 0 1
+    // @User: Standard
+    AP_GROUPINFO("PROP_PIT_COMP", 37, ParametersG2, propwash_pitch_comp, 0.3),
+
+    // @Param: PROP_YAW_COMP
+    // @DisplayName: Propwash yaw compensation
+    // @Description: This is a scaling factor applied to the theoretical calculation of rudder effectiveness variation with propeller thrust. The PROP_DSKLD parameter must be set for this to work,
+    // @Range: 0 1
+    // @User: Standard
+    AP_GROUPINFO("PROP_YAW_COMP", 38, ParametersG2, propwash_yaw_comp, 0.5),
```

Comment:
I wonder if defaulting to different values for different axes is a bit confusing, maybe they should all be 0.5? Presumably the user should go and set them all anyway.

Reply:
These are what I expect to be typical values for conventional planes with elevator and rudder behind the prop, however am also OK with setting them to 0 as defaults.

Reply:
I've set the defaults to 0( as there is not a consensus yet on typical values) and updated the tuning advice in the wiki PR https://github.com/ArduPilot/ardupilot_wiki/pull/4733